Sources/SpeziOnboarding/OnboardingFlow/OnboardingFlowViewCollection.swift

```diff
@@ -0,0 +1,22 @@
+//
+// This source file is part of the Stanford Spezi open-source project
+//
+// SPDX-FileCopyrightText: 2022 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import Foundation
+import SwiftUI
+
+/// A ``OnboardingFlowViewCollection`` defines a collection of SwiftUI `View`s that are defined with an ``OnboardingStack``.
+///
+/// You can not create a ``OnboardingFlowViewCollection`` yourself. Please use the ``OnboardingStack`` that internally creates a ``OnboardingFlowViewCollection`` with the passed views.
+public class OnboardingFlowViewCollection: ObservableObject {
```

Comment:
One thing to more explicitly hide it from e.g. autocompletion and the documentation generation would be adding an underscore to the type:
```suggestion
public class _OnboardingFlowViewCollection: ObservableObject {
```

Reply:
Great tip, incorporated it!